libindex/options.go

```diff
@@ -11,6 +11,11 @@ const (
 	DefaultLayerScanConcurrency = 10
 )
 
+// Opts is a compatibility alias.
+//
+// Deprecated: Calling code should use the new name.
+type Opts = Options
```

Comment:
is this needed? isn't it already `Options`?

Reply:
Yeah, this doesn't do anything here. I think this is confusing `indexer.Opts` and `libindex.Options`.

Reply:
:+1: